bigquery_etl/metadata/validate_metadata.py

```diff
@@ -215,6 +218,148 @@ def validate_shredder_mitigation(query_dir, metadata):
     return True
 
 
+def count_schema_fields(schema):
+    """Return the count of fields and how many have descriptions in a given BigQuery schema."""
+    if not schema or not isinstance(schema, (list, tuple)):
+        return (0, 0)
+
+    fields = descriptions = 0
+    base = list(schema)
+    while base:
+        field = base.pop()
+        fields += 1
+        if getattr(field, "description", None):
+            descriptions += 1
+        if hasattr(field, "fields") and field.fields:
+            base.extend(field.fields)
+    return fields, descriptions
+
+
+def find_bigeye_checks(query_path):
+    """Return True if bigeye metrics are present or detail of missing bigeye metrics or file."""
+    predefined_metrics = {"FRESHNESS", "VOLUME"}
+    bigeye_file_path = os.path.join(query_path, BIGEYE_PREDEFINED_FILE)
+    found_types = set()
+
+    if not os.path.exists(bigeye_file_path):
+        return False
+    with open(bigeye_file_path, "r") as f:
+        try:
+            content = yaml.safe_load(f)
+        except yaml.YAMLError:
+            click.echo("ERROR: Failed to read BigEye config.")
+            return False
+
+    stack = [content]
+    while stack:
+        obj = stack.pop()
+        if isinstance(obj, dict):
+            for v in obj.values():
+                stack.append(v)
+        elif isinstance(obj, list):
+            stack.extend(obj)
+        elif isinstance(obj, str):
+            if obj in predefined_metrics:
+                found_types.add(obj)
+
+    missing = predefined_metrics - found_types
+
+    if any(metric in missing for metric in predefined_metrics):
+        click.echo(f"ERROR: Missing Bigeye metrics: {missing}.")
+        return False
+    return True
+
+
+def validate_asset_level(query_dir, metadata):
```

Comment:
how about missing owners?

Reply:
In my opinion, it makes sense to require owners, at the minimum for level Gold.
Could you add it to [Requirements for each level](https://docs.google.com/document/d/1K8f4Jm0pRJOTbQpUs4KALnPmUSWg_mmJ1L_1484yEeg/edit?tab=t.0#heading=h.pu9y4swccxnc) for which levels you think it makes sense?